observation_portal/requestgroups/serializers.py

```diff
@@ -898,6 +898,9 @@ class CadenceRequestGroupSerializer(RequestGroupSerializer):
     # override the validate method from the RequestGroupSerializer and use the Cadence Request serializer to
     # validate all of the Cadence Requests
     def validate(self, data):
+        if len(data['requests']) > 1:
```

Comment:
I think we might as well check here that there is at least one request as well

Reply:
So I initially did that, but in my testing I noticed that it's actually enforced at the field level because of the `many=True` kwarg passed into the serializer constructor.

So when sending a requestgroup with `"request": []` with the code as-is

we get

```
{
  "requests": [
    "You must specify at least 1 request"
  ]
}
```

I am happy to add a check like 

```
        if len(data['requests']) == 0 or len(data['requests']) > 1:
            raise ValidationError('Cadence requestgroups must contain a single request')
```

but in the case of an empty list we don't actually get to this condition

Reply:
In any case,, I'll add a test to make sure we don't error out with an empty request list.

Reply:
Ah, ok, that works then! yeah adding a test sounds good